frontend/src/api/hooks/team/actions/useTeamEdit.ts

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios'
+import { TeamEditDto, TeamResponses } from '../../../../util/views/team.view'
+import { useState } from 'react'
+
+export const useTeamEdit = (onResponse: (response: TeamResponses) => void) => {
+  const [loading, setLoading] = useState(false)
+  const [error, setError] = useState<Error>()
+  const teamEdit = (dto: TeamEditDto) => {
+    setLoading(true)
+    const data = new FormData()
+    data.append('description', dto.description)
+    if (dto.logo) {
+      data.append('logo', dto.logo)
+    }
+    axios
+      .post<TeamResponses>(`/api/team/edit`, data)
+      .then((res) => {
+        onResponse(res.data)
+      })
+      .catch((err) => {
+        setError(err)
+        onResponse(TeamResponses.ERROR)
+      })
+      .finally(() => {
+        setLoading(false)
+      })
+  }
+  return { teamEditLoading: loading, teamEditError: error, teamEdit }
+}
```

Comment:
A react-query, amit a többi network kéréshez használunk a loading meg error stateket kezeli magától. Ez is tök jól működik, de egy refaktor majd a jövőben ráfér, hogy egységes legyen a kódbázis

Reply:
igazából a csak lemásoltam az egyik meglévő teames hookot, az összes ilyen

Reply:
van valamennyi tech debt az biztos, lehet érdemes lassan időt szánni rá

Reply:
ohh oké, akkor ezek szerint eddig se voltunk konzisztensek :D